data_model/master/clusters/Identify.xml

```diff
@@ -122,7 +122,7 @@ Davis, CA 95616, USA
   <attributes>
     <attribute id="0x0000" name="IdentifyTime" type="uint16" default="0">
       <access read="true" write="true" readPrivilege="view" writePrivilege="operate"/>
-      <quality changeOmitted="false" nullable="false" scene="false" persistence="volatile" reportable="false"/>
+      <quality changeOmitted="false" nullable="false" scene="false" persistence="volatile" reportable="false" sourceAttribution="false" quieterReporting="true"/>
```

Comment:
Properly a stupid question, but I assume the reason why this attribute has the quality tag and e.g. IdentifyType is because this (IdentifyTime) actually has a quality defined, while IdentifyType does not, but if does not seem like F is being marked as a thing in the quality tag?

Reply:
I think so. I'm not sure quite why all the attributes are explicitly added with the defaults when the tags are not, but ... whatever. We can always change this in alchemy once we swap over if we want to enforce something different in the xml schema.